<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureWeb</class>
 <widget class="QWidget" name="ConfigureWeb">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBoxWebConfig">
       <property name="title">
        <string>Citra Account</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutCitraWebService">
        <item>
         <widget class="QLabel" name="web_credentials_disclaimer">
          <property name="text">
           <string>This is required to host public rooms.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayoutCitraUsername">
          <item row="0" column="1" colspan="3">
           <widget class="QLabel" name="username">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_token">
            <property name="text">
             <string>Token: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QLineEdit" name="edit_token">
            <property name="maxLength">
             <number>80</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_token_verified"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="web_signup_link">
            <property name="text">
             <string>Sign up</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_username">
            <property name="text">
             <string>Username: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="button_verify_login">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Verify</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="web_token_info_link">
            <property name="text">
             <string>What is my token?</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_disable_info">
       <property name="text">
        <string>Citra account configuration can only be changed when a public room isn't being hosted.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Citra Valentin API URL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cv_web_api_url">
         <property name="text">
          <string>https://cv-aadb.glitch.me</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="enable_discord_rp">
       <property name="text">
        <string>Enable Discord Rich Presence</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="discord_rp_show_settings" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>16</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="discord_rp_show_game_name">
          <property name="text">
           <string>Show Game Name</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="discord_rp_show_room_information">
          <property name="text">
           <string>Show Room Information</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connect_discord">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Required for sending logs and screenshots to Citra Valentin's Discord server.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Connect Discord to Citra Valentin</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
